### DISCUSSION TITLE
Principle: An open source project that is a component of a commercial offering must provide tangible value in its own right

### DISCUSSION THREAD

**Discussion by ColinEberhardt** [finos/technical-oversight-committee#21] [General]
Why? There are various models (e.g. open-core) whereby parts of a commercial project are open-sourced, delivering genuine value to the community. These component can be used freely without the need to purchase the associated product. However, there can be instances where the open source component is not useable or valuable of itself, and the act of open sourcing is primarily being used as a sales channel. When evaluating open source components associated with commercial products, we need to ensure that they deliver value without the need to purchase the associated commercial product.

**Comment by ColinEberhardt** [comment]
Just updated, I feel this is a 'must'

> Principle: An open source project that is a component of a commercial offering ~should~ must provide tangible value in its own right

**Comment by rikoe** [comment]
How do we define "tangible value"? Is this by its very nature subjective?

**Reply by ColinEberhardt** [reply]
Yes, I think it is going to be subjective - there are so many different ways that you can mix open source and proprietary code.
